src/openrct2/windows/footpath_scenery.c

```diff
@@ -0,0 +1,484 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
@IntelOrca New C file, but uses spaces for indentation. Not sure if we want C to be spaces only, or we don't really care.

@janisozaur Is the copyright date mandatory? Seems like a hassle to update each year.